src/libxtp/gwbse/gwbse.cc

```diff
@@ -319,10 +319,11 @@ void GWBSE::Initialize(tools::Property& options) {
       _gwopt.g_sc_max_iterations);  // convergence criteria for qp iteration
                                     // [Hartree]]
 
-  _gwopt.gw_sc_max_iterations = options.ifExistsReturnElseReturnDefault<Index>(
-      key + ".gw_sc_max_iterations",
-      _gwopt.gw_sc_max_iterations);  // convergence criteria for qp iteration
-                                     // [Hartree]]
+  if (mode == "evGW") {
```

Comment:
why this change?

Reply:
Because it seems you could have asked for G0W0, but if you later change gw_sc_max_iterations you override this.

Reply:
okay